ticdc/ticdc-changefeed-overview.md

```diff
@@ -11,26 +11,30 @@ A changefeed is a replication task in TiCDC, which replicates the data change lo
+- `Warning`: The replication task returns an error. The replication cannot continue due to some recoverable errors. The changefeed in this state keeps trying to resume until the state transfers to `Normal`. The maximum retry time is 30 minutes. If it exceeds this time, the changefeed enters a failed state. The changefeed in this state blocks GC operations.
 - `Finished`: The replication task is finished and has reached the preset `TargetTs`. The changefeed in this state does not block GC operations.
-- `Failed`: The replication task fails. Due to some unrecoverable errors, the replication task cannot resume and cannot be recovered. TiCDC retains the data associated with the failed changefeed for 24 hours, preventing it from being garbage collected.
+- `Failed`: The replication task fails. Due to some unrecoverable errors, the replication task cannot resume and cannot be recovered. To give you enough time to handle the failure, the changefeed in this state blocks GC progress. The duration of the blockage is specified by the `gc-ttl` parameter, with a default value of 24 hours.
+
+> **Note:**
+>
+> If the changefeed encounters errors with error codes `ErrGCTTLExceeded`, `ErrSnapshotLostByGC`, or `ErrStartTsBeforeGC`, it does not block GC progress.
 
 The numbers in the preceding state transfer diagram are described as follows.
 
 - ① Run the `changefeed pause` command.
 - ② Run the `changefeed resume` command to resume the replication task.
-- ③ Recoverable errors occur during the `changefeed` operation, and the operation is resumed automatically.
-- ④ Run the `changefeed resume` command to resume the replication task.
-- ⑤ Unrecoverable errors occur during the `changefeed` operation.
-- ⑥ `changefeed` has reached the preset `TargetTs`, and the replication is automatically stopped.
-- ⑦ `changefeed` suspended longer than the duration specified by `gc-ttl`, and cannot be resumed.
-- ⑧ `changefeed` experienced an unrecoverable error when trying to execute automatic recovery.
+- ③ Recoverable errors occur during the `changefeed` operation, and the operation is retried automatically.
+- ④ The changefeed automatic retry succeeds, and `checkpoint-ts` continues to advance.
+- ⑤ The changefeed automatic retry exceeds 30 minutes and fails. The changefeed enters the failed state. At this time, the changefeed continues to block upstream GC for a duration specified by `gc-ttl`.
+- ⑥ The changefeed encounters an unrecoverable error and directly enters the failed state. At this time, the changefeed continues to block upstream GC for a duration specified by `gc-ttl`.
+- ⑦ The replication progress of the changefeed reaches the value set by `target-ts`, and the replication is completed.
+- ⑧ The changefeed has been stalled for a duration longer than the value specified by `gc-ttl`, encounters GC advancement errors, and cannot be recovered.
```

Comment:
```suggestion
- ⑧ The changefeed has been suspended for a duration longer than the value specified by `gc-ttl`, thus encountering GC advancement errors, and cannot be resumed.
```

Reply:
the suggestion is based on https://docs.pingcap.com/tidb/dev/ticdc-faq#what-is-gc-ttl-in-ticdc.